index.mjs

```diff
@@ -21,7 +42,8 @@ export {
 // https://en.wikipedia.org/wiki/Longest_common_subsequence_problem
 //
 // Expects two arrays, finds longest common sequence
-function LCS(buffer1, buffer2) {
+function LCS(buffer1, buffer2, timeout) {
+  timeout = getAssertTimeout(timeout);
```

Comment:
```suggestion
  assertTimeout = getAssertTimeout(timeout);
```
I think this will help with readability (later in the code when I see `timeout()` it seems as though we are saying we timed out, not checking)
also, not loving overriding input arguments